### ISSUE TITLE
Make favicon

### ISSUE THREAD

**Issue by JuliaJRH** [JuliaJRH/journosalarysharer#9]
Replace mystery favicon

**Comment by JuliaJRH** [comment]
![salarysharerfavicon](https://cloud.githubusercontent.com/assets/10747135/8940601/cb7f54f2-3538-11e5-8ee3-2da61309c82c.png)

**Comment by JuliaJRH** [comment]
In the gh-pages branch as SalarySharerFavicon.png

**Comment by jueyang** [comment]
Could you change the favicon to 16x16 pixels? That's the standard dimension.

Also, when you are at it, make a logo (probably just need some dimension change) that can go next to the site title. That'd be a nice touch.

**Comment by JuliaJRH** [comment]
There's a favicon at 16x16 there now. I made it 64x64 because when I researched what size to make it, that's what I found as the suggested size here: http://blog.teamtreehouse.com/how-to-make-a-favicon 

As for the logo, I'm not prepared to make a logo for this at this time. I like the design without any logo, and if I'm going to make a logo, I need to set aside the time to do it right and get feedback, which I don't have time for in the next 36 hours.
